dts/arm/st/f1/stm32f103X8.dtsi

```diff
@@ -8,15 +8,19 @@
 #include <st/f1/stm32f1.dtsi>
 
 / {
-	flash0: flash@8000000 {
-		reg = <0x08000000 DT_SIZE_K(64)>;
-	};
-
 	sram0: memory@20000000 {
 		reg = <0x20000000 DT_SIZE_K(20)>;
 	};
 
 	soc {
+
+		flash-controller@40022000 {
+			flash0: flash@8000000 {
+				reg = <0x08000000 DT_SIZE_K(64)>;
+				erase-block-size = <DT_SIZE_K(1)>;
```

Comment:
Has this device regular flash page layout?

Reply:
regular?

Reply:
== each flash page has the same size.

Reply:
According to the manual there are two sizes, small and medium capacity is 1k large capacity and Internet products are 2k.

Reply:
looks like almost regular, but part of the flash mapped here (the main memory) has unified page size, so looks like this is OK